src/ralph/assets/models/mixins.py

```diff
@@ -64,3 +60,11 @@ def save(self, update_last_seen=False, *args, **kwargs):
         if update_last_seen:
             self.last_seen = timezone.now()
         super(LastSeenMixin, self).save(*args, **kwargs)
+
+
+class AdminAbsoluteUrlMixin(object):
+    def get_absolute_url(self):
+        info = self._meta.app_label, self._meta.model_name
+        return reverse(
+            'admin:{}_{}_change'.format(*info), args=(self.pk,)
```

Comment:
could it be simply:
`'admin:{}_{}_change'.format(self._meta.app_label, self._meta.model_name), args=(self.pk,)`
?